src/features/logging/instrument/index.js

```diff
@@ -0,0 +1,19 @@
+import { stringify } from '../../../common/util/stringify'
+import { InstrumentBase } from '../../utils/instrument-base'
+import { FEATURE_NAME } from '../constants'
+import { bufferLog } from '../shared/utils'
+
+export class Instrument extends InstrumentBase {
+  static featureName = FEATURE_NAME
+  constructor (agentIdentifier, aggregator, auto = true) {
+    super(agentIdentifier, aggregator, FEATURE_NAME, auto)
+
+    const instanceEE = this.ee
+    /** emitted by wrap-logger function */
+    this.ee.on('wrap-logger-end', function handleLog ([message, ...args]) {
+      const { level } = this
+      bufferLog(instanceEE, message, { ...(!!args.length && { customAttributes: stringify(args) }) }, level)
```

Comment:
```suggestion
      bufferLog(instanceEE, message, args.length ? { customAttributes: stringify(args) } : {}, level)
```
is more readable + I'm a confused why the customAttributes/Log().attributes which is an object of args have a key of `customAttributes` with value of additional args stringified. Is that supposed to be the case?

Reply:
I realized this inconsistency in a subsequent PR and thus this is fixed [here](https://github.com/newrelic/newrelic-browser-agent/pull/1061/files#diff-e37844b7cdcf3225ff308f042059b5b5c9d92ecdf60b9cc8c76fe3d2789c7a5cR12-R15) to match basically what you wrote, it aligns more nicely for sure.  unfortunate pattern of this split PR technique :/